packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.component.tsx

```diff
@@ -61,19 +59,37 @@ const VitalsHeader: React.FC<VitalsHeaderProps> = ({ patientUuid }) => {
     const vitalsOverdue = hasActiveVisit && !vitalsTakenToday;
     const now = dayjs();
     const vitalsOverdueDayCount = Math.round(dayjs.duration(now.diff(latestVitals?.date)).asDays());
-    let overdueVitalsTagContent = '';
+
+    let overdueVitalsTagContent: React.ReactNode = 'null';
 
     switch (true) {
       case vitalsOverdueDayCount >= 1 && vitalsOverdueDayCount <= 7:
```

Comment:
It might make more sense to define this as `>= 1 && < 7`... I think that's a more intuitive meaning of "over a week". Ditto `>= 7 && < 14` for "over one week". 

(Also, what's with this `switch(true)` construct? I think the `if...else` version is actually cleaner unless we were treating it as an expression or something?)